specification/vmwarecloudsimple/resource-manager/Microsoft.VMwareCloudSimple/stable/2019-04-01/vmwarecloudsimple.json

```diff
@@ -0,0 +1,3604 @@
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/microsoft.vmwarecloudsimple/operations": {
+      "get": {
+        "description": "Return list of operations",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "tags": [
+          "operations"
+        ],
+        "summary": "Implements list of available operations",
+        "operationId": "availableOperations_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/AvailableOperationsListResponse"
+            }
+          },
+          "default": {
+            "description": "BadRequest",
+            "schema": {
+              "$ref": "#/definitions/CSRPError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ListOperations": {
+            "$ref": "./examples/ListOperations.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/microsoft.vmwarecloudsimple/dedicatedcloudnodes": {
+      "get": {
+        "description": "Returns list of dedicate cloud nodes within subscription",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "tags": [
+          "dedicatedCloudNode"
+        ],
+        "summary": "Implements list of dedicated cloud nodes within subscription method",
+        "operationId": "dedicatedCloudNode_ListBySubscription",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "type": "string",
+            "description": "The filter to apply on the list operation",
+            "name": "$filter",
+            "in": "query"
+          },
+          {
+            "type": "integer",
+            "format": "int32",
+            "description": "The maximum number of record sets to return",
+            "name": "$top",
+            "in": "query"
+          },
+          {
+            "type": "string",
+            "description": "to be used by nextLink implementation",
+            "name": "$skipToken",
+            "in": "query"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/DedicatedCloudNodeListResponse"
+            }
+          },
+          "404": {
```

Comment:
you dont need to list the 404 response separately. It can be covered via the default response.